libs/designsystem/src/lib/components/page/page.component.spec.ts

```diff
@@ -52,11 +54,15 @@ describe('PageComponent', () => {
       TestHelper.ionicModuleForTest,
       RouterTestingModule.withRoutes([
         {
-          path: '',
+          path: pageUrl,
           component: PageComponent,
         },
         {
-          path: 'someUrl',
+          path: firstOtherUrl,
+          component: DummyComponent,
+        },
+        {
+          path: secondOtherUrl,
```

Comment:
Could these be Page components instead, so we make sure to test the stacked behavior? We would have to refactor the enter/leave tests below a tiny bit to account for additional enter/leaves being called when navigating.

Reply:
Good point - done